docs/001_develop/03_client-capabilities/008_interaction/accordian.mdx

```diff
@@ -0,0 +1,320 @@
+---
+title: 'Accordion'
+sidebar_label: 'Accodion'
+id: client-interaction-accordion
+keywords: [ interaction, interactive, accordion, accordian, disclosure, expand, hide, collapse, fold]
+tags:
+- interaction
+- interactive
+- accordion
+- accordian
+- disclosure
+- expand
+- hide
+- collapse
+- fold
+sidebar_position: 8
+---
+
+import AccordionDemo from '/examples/ui/client-capabilities/interaction/accordion.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Accordions are a type of disclosure that allows users to expand or collapse sections of content on a page.
```

Comment:
Suggestion: reword this to remove disclosure, as its not obvious what that means to a lame.
I think this covers it with no ambiguiy:
Accordions allow users to expand or collapse sections of content on a page.